Cargo.toml

```diff
@@ -118,10 +121,13 @@ ef-tests = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311d
 reth-ethereum-primitives = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf" }
 reth-stateless = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf" }
 reth-rpc-api = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf" }
+reth-primitives-traits = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf" }
```

Comment:
I moved back to our current reth-fork, so now creating the `BincodeBlock` in this new `ere-guests/lib` crate.

The idea is that this crate will be part of both the main workspace of this repo, and the `ere-guests` workspace. But the latter will happen when we do the `ere`-Dockerization change.